central/policy/service/customresource/custom_resource.go

```diff
@@ -0,0 +1,42 @@
+package customresource
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/utils"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"gopkg.in/yaml.v3"
+)
+
+// CustomResource represents the structure for a SecurityPolicy Kubernetes Custom Resource.
+type CustomResource struct {
+	APIVersion string                 `yaml:"apiVersion"`
+	Kind       string                 `yaml:"kind"`
+	Metadata   map[string]interface{} `yaml:"metadata,omitempty"`
+	Spec       *Policy                `yaml:"spec"`
```

Comment:
We are renaming to SecurityPolicy

Reply:
Its yaml representation is still "spec", so it does not matter what is named here. I see the current naming is SecurityPolicySpec.